test/gas-report/poolsize.js

```diff
@@ -395,5 +395,5 @@ contract("transcoder pool size gas report", accounts => {
 
     testWithPoolSize(100)
     testWithPoolSize(200)
-    testWithPoolSize(300)
+    // testWithPoolSize(300)
```

Comment:
This was re-commented (it wasn't prior to this commit) due to coverage being flaky on this ? https://github.com/livepeer/protocol/issues/393

Reply:
Fixed in 654de5b